src/exporter/DaemonMetricCollector.cc

```diff
@@ -137,6 +139,7 @@ void DaemonMetricCollector::dump_asok_metrics() {
       }
     }
   }
+  std::cout << "Perf counters retrieved for " << clients.size() << " daemons." << std::endl;
```

Comment:
What about summarizing the successful ones vs the total configured?

Reply:
```suggestion
  dout(level) << "Perf counters retrieved for " << clients.size() << " daemons." << dendl;
```